### ISSUE TITLE
Menu is hidden for small JSONs

### ISSUE THREAD

**Issue by sevdog** [nnseva/django-jsoneditor#48]
If a small JSON (with very few keys) is provided to this widget the menu gets hidden by Django's containing elements overflow style (`.form-row` CSS class has `overflow: hidden`).

![Schermata del 2019-11-07 12-29-52](https://user-images.githubusercontent.com/13779643/68385637-80ee3280-015a-11ea-8b85-fc8035ce6eb4.png)
![image](https://user-images.githubusercontent.com/13779643/68386060-8a2bcf00-015b-11ea-8003-5d117787c4bb.png)

This makes menu unusable.

It could be avoided by using `position: fixed` but this will need extra code to position it or a padding could be added to prevent menu from overflowing external row.

Also a custom class or rule could be added to avoid this.

**Comment by ghost** [comment]
You can add `min-height: 500px;` to the selector `div.jsoneditor`,but if you add too many columns, the final menu will still overflow.

**Comment by nnseva** [comment]
Please review the problem on your side @sevdog, does it appear now? Some changes were made in the underlining JS package, it may solve your issue.

If the problem still appears, please report here your python and Django version, settings regarding django-jsoneditor, and code initiating it in your case.

**Comment by sevdog** [comment]
@nnseva even if the original JS library was updated I still got issues with sizing:
![image](https://user-images.githubusercontent.com/13779643/74846068-315dfc00-5330-11ea-9610-0abcd589ada0.png)


This is my environment:
- Python 3.6.10
- Django 2.2.10
- django-jsoneditor 0.1.4
```
# JS/CSS pointing to latest release tag
JSON_EDITOR_JS = 'https://cdnjs.cloudflare.com/ajax/libs/jsoneditor/8.6.1/jsoneditor.min.js'
JSON_EDITOR_CSS = 'https://cdnjs.cloudflare.com/ajax/libs/jsoneditor/8.6.1/jsoneditor.min.css'
```

As I said before this is caused by `.form-row` class which wraps around the field with `overflow: hidden`, if you manage to remove or change that class there will be no problem.

**Comment by nnseva** [comment]
I've checked with the contributed jsoneditor JS code and found that the picture is fine in this case.

![image](https://user-images.githubusercontent.com/496812/74911667-20fc5e80-53ce-11ea-9903-5750c5ddfdf2.png)

The `jsoneditor` JS code referenced in your settings file causes the problem. Just remove or comment out `JSON_EDITOR_*` lines in your settings, and don't forget to clean up the browser cache.

**Comment by nnseva** [comment]
sorry, I answered wrongly, checking ...

**Comment by nnseva** [comment]
Thanx a lot @sevdog, a new fixed version of the package has been uploaded.

**Comment by sevdog** [comment]
Commit e5e9ee1cae192a82edd303740ca0ef9dad086f72 has broken this feature again.

I know that  #56 should be taken into account, but at this point the "override" of CSS rule should be done via JavaScript since JSONEditor menu is useless for small JSONs.

I mean: a global override has too many side effects, only remove or fix overflow for JSON rows.

**Comment by nnseva** [comment]
Hi @sevdog ,

starting from the commit https://github.com/nnseva/django-jsoneditor/commit/ba2ae948a3c5959a6eeb2fa1cd87dc9a25329987 the json editor window is resized. You always can resize it to fit the menu if necessary.

So the previous override workaround which breaks some other Django features was eliminated.

**Comment by sevdog** [comment]
Thanks @nnseva for the explaination. Although IMO having a context-menu half-hidden by default is very unpleasant in term of UX, I acknowledge that beeing able to expand the field can work-around the issue.

However if the field get expanded over its containing node (which may happen since the "drag" event does not take care of bounds) then is not possible to change its size anymore. This prevents users from expand it enough to see the full menu (and possibly part of the JSON).

Here is an example (watch for the right border where the containing node ends and the `overflow: hidden` rule takes effect):
![jsoviewer](https://user-images.githubusercontent.com/13779643/145360705-abc487c1-1a66-4cca-81e1-92f402b95d94.gif)

I dunno if this is eligible for an appropriare issue or is a bug of the JS JSONEditor itself.

**Comment by nnseva** [comment]
Thanx @sevdog , fixed in 77583a45da52907310a5ce37672b24d1c4255a58
